src/RenderPropsComponents/HeaderButton/test.tsx

```diff
@@ -1,11 +1,15 @@
 import { shallow } from 'enzyme'
 import React from 'react'
 
-import { BLOCK_CLASS_NAME, NEXT_MONTH_TITLE, PREV_MONTH_TITLE } from '../consts'
-import HeaderButton from '../header_button'
+import {
+  BLOCK_CLASS_NAME,
+  NEXT_MONTH_TITLE,
+  PREV_MONTH_TITLE
+} from '../../calendar/consts'
+import HeaderButton from '.'
 
 describe('HeaderButton', () => {
-  let props, wrapper
+  let props: any, wrapper: any
```

Comment:
`props` -> ComponentProps<typeof HeaderButton>

Reply:
for now it should be fine 😄